test/e2e/test_setting.py

```diff
@@ -212,8 +243,19 @@ def test_create_item_after_spares_definition_set_with_multiple_types(self):
         self.post_item_in_system_with_type_id(SYSTEM_TYPE_GET_DATA_STORAGE["id"])
         self.check_catalogue_item_spares([1, 3])
 
+        # Delete the rule that allows items to be created in Operational systems to avoid conflicts with other tests
+        self.delete_rule(custom_creation_rule_id)
```

Comment:
Is it not worth doing the same thing in the conf test to remove predefined rules like the usage status one? Right now if the test fails it wont reset and it will be stuck there. I think its just the setup.mongodb would need predefined ids to do this.